cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -601,9 +602,248 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         }
       }
 
+      section("The dbms `EXECUTE PROCEDURE` privileges", "administration-security-administration-dbms-privileges-execute-procedure", "enterprise-edition") {
+        initQueries(
+          "CREATE ROLE procedureExecutor",
+          "CREATE ROLE deniedProcedureExecutor",
+          "CREATE ROLE boostedProcedureExecutor",
+          "CREATE ROLE deniedBoostedProcedureExecutor1",
+          "CREATE ROLE deniedBoostedProcedureExecutor2",
+          "CREATE ROLE deniedBoostedProcedureExecutor3",
+          "CREATE ROLE deniedBoostedProcedureExecutor4",
+          "CREATE ROLE adminProcedureExecutor",
+          "CREATE ROLE procedureGlobbing1",
+          "CREATE ROLE procedureGlobbing2",
+          "CREATE ROLE procedureGlobbing3",
+          "CREATE ROLE procedureGlobbing4",
+          "CREATE ROLE procedureGlobbing5"
+        )
+        p("The dbms privileges for procedure execution are assignable using Cypher administrative commands. They can be granted, denied and revoked like other privileges.")
+        p("include::dbms/execute-procedure-syntax.asciidoc[]")
+
+        section("The `EXECUTE PROCEDURE` privilege", "execute-procedure-subsection", "enterprise-edition") {
+          p(
+            """The ability to execute a procedure can be granted via the `EXECUTE PROCEDURE` privilege.
+              |A user with this privilege is allowed to execute the procedures matched by the <<procedure-name-globbing, name-globbing>>.
+              |The following query shows an example of how to grant this privilege:""".stripMargin)
+          query("GRANT EXECUTE PROCEDURE db.schema.* ON DBMS TO procedureExecutor", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureExecutor' can then run any procedure in the `db.schema` namespace. The procedure will be run using the users own privileges.")
```

Comment:
Did you intend for this sentence to appear between the query and result? 
![Screenshot 2020-09-23 at 20 18 38](https://user-images.githubusercontent.com/12032446/94059188-fc3fed80-fdd9-11ea-8b6b-cfab467fa7ce.png)